config/log4j.properties

```diff
@@ -13,47 +13,59 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-log4j.rootLogger=INFO, stdout 
+log4j.rootLogger=INFO, stdout
 
 log4j.appender.stdout=org.apache.log4j.ConsoleAppender
 log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c)%n
 
-log4j.appender.kafkaAppender=org.apache.log4j.DailyRollingFileAppender
-log4j.appender.kafkaAppender.DatePattern='.'yyyy-MM-dd-HH
+log4j.appender.kafkaAppender=org.apache.log4j.log4j.RollingFileAppender
 log4j.appender.kafkaAppender.File=${kafka.logs.dir}/server.log
 log4j.appender.kafkaAppender.layout=org.apache.log4j.PatternLayout
 log4j.appender.kafkaAppender.layout.ConversionPattern=[%d] %p %m (%c)%n
+log4j.appender.kafkaAppender.Append=true
+log4j.appender.kafkaAppender.MaxBackupIndex=10
+log4j.appender.kafkaAppender.MaxFileSize=100MB
```

Comment:
We run Kafka on Docker containers with 12GB of allocated disk, including the OS, and since we have multiple logs and there's no compression(that would be a nice improvement), that may add up to filling up the disk. I know that's quite small, and that disk is cheap nowadays, but it also matters if someone is actually looking at those logs on the machines, otherwise they are useless.

The question is if this config is supposed to be production-ready as it is, or just an example that people are supposed to adjust to their production installations. I for one would prefer to have sane defaults which I don't need to override, but I don't mind much if I need to change these values for our production environment.